src/composables/GetBlog.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios"
+
+
+const Blogs = () => {
+
+  const getBlog = async () => {
+
+    try {
+      const response = await axios.get("http://localhost:3000/posts");
```

Comment:
Just a note. Please read about Axios Defaults. Since we normally communicate with a single backend most of the time. We can use axios base url default to set the base url for all the rest of api calls. 

https://axios-http.com/docs/config_defaults